apps/ledger-live-mobile/src/components/RequiresBLE/RequiresBluetoothEnabled.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState, useEffect } from "react";
+import { Observable } from "rxjs";
+import TransportBLE from "../../react-native-hw-transport-ble";
+import { usePromptBluetoothCallbackWithState } from "../../logic/usePromptBluetoothCallback";
+import BluetoothDisabled from "./BluetoothDisabled";
+
+type Props = {
+  children?: React.ReactNode;
+};
+
+const RequiresBluetoothEnabled: React.FC<Props> = ({ children }) => {
+  const [type, setType] = useState<string>("Unknown");
+
+  const { prompt, bluetoothPromptedOnce, bluetoothPromptSucceeded } =
+    usePromptBluetoothCallbackWithState();
+
+  useEffect(() => {
+    prompt();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  useEffect(() => {
+    const sub = Observable.create(TransportBLE.observeState).subscribe({
```

Comment:
`create` method is deprecated (i know it's not you 🙃)

But it can easily be replaced by:
```
const sub = new Observable(TransportBLE.observeState).subscribe({ .... })
```

Reply:
ah thanks, I'll do that!